pzsd_bot/settings.py

```diff
@@ -41,11 +49,18 @@ class _DB(EnvSettings):
     pghost: str = "localhost"
     pgport: int = 5432
     pgdatabase: str = "pzsd"
+    testing: bool = False
+
+    @property
+    def connection_str(self) -> str:
+        if self.testing:
```

Comment:
applications shouldn't be environment aware. you could either pass in the whole connection string or you could rename this to use_sqlite